src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.test.tsx

```diff
@@ -11,17 +11,19 @@ jest.mock('react-router-dom', () => ({
 
 const { Default } = composeStories(stories);
 
```

Comment:
right now these are UI tests. They are not really testing the flow. They are testing the render utilities of the underlying children components of the main component ProfilePageWalletConnected. The tests you are mentioning here are e2e tests. They are a bit harder to setup since we will need fixed wallets with fixed badges and so on.

Reply:
got it, that makes sense. keen to setup some of these e2e tests!